docs/Contributing.asciidoc

```diff
@@ -203,7 +203,9 @@ meta-package which pulls all required dependencies for development.
 
 [[setup-postgresql]]
 === Setting up the PostgreSQL database
-One also needs to setup a PostgreSQL database for openQA manually owned by your regular user:
+One also needs to setup a PostgreSQL database for openQA manually owned by
+your regular user. This can be done with the script `openqa-setup-db` or
```

Comment:
The script lacks steps 1. and 2. because if someone only checked things out via Git (as this particular part of the *contributing* documentation suggests) PostgreSQL might be not be installed and or started.

Besides, this is the section of the documentation which explains how to develop openQA itself. At this point one likely wants to use the regular user to run the services. In this case the sentence " where `your_username` must be the same as the UNIX user you start your local openQA instance with."  of the steps below should *not* be ignored. However, a user following the documentation who decides to use the script would likely ignore this sentence and not pass the arguments for the db user and name to the script.

So in this form mentioning the script here makes things worse.

Reply:
It would make more sense to me to mention the script as an alternative to the actual step 4/5 i.e. if these were merged into something like

    Create the user: ...... The script `openqa-setup-db` can be used to the same effect.

Reply:
good point. Sorry, didn't pay much attention to this today. I have followed the suggestion by @kalikiana and mentioned just after the instructions how the script `openqa-setup-db` covers parts of these steps.